plugins/Speech/__init__.py

```diff
@@ -383,7 +427,7 @@ def OnButton(event):
             (sizer2, 0, wx.EXPAND|wx.ALL, 5),
         )
         ACV = wx.ALIGN_CENTER_VERTICAL
-        sizer3 = wx.FlexGridSizer(0, 2, 5, 5)
+        sizer3 = wx.FlexGridSizer(0, 0, 5, 5)
```

Comment:
Leave the column count at 2 else all widget will be in one row.

![eg-speech](https://cloud.githubusercontent.com/assets/6463113/21866446/9b002ff2-d84a-11e6-8c3b-95c265bafc1d.png)